resources/definitions/mixware_hyper_s.def.json

```diff
@@ -299,7 +290,7 @@
             "value": "wall_line_width_0"
         },
         "support_z_distance": {
-            "value": "layer_height if layer_height >= 0.16 else layer_height * 2"
+            "value": "layer_height"
```

Comment:
Are you sure about this change? I've personally noticed that 2x the layerheight is too high if you're printing with 0.2 layerheight.